news/4.33/pde.html

```diff
@@ -46,6 +46,21 @@ <h2>Plug-in Development Environment</h2>
   <tr>
     <td id="editors" class="section" colspan="2"><h2>Editors</h2></td>
   </tr>
+  <tr id="osgirepositories">
+    <!-- https://github.com/eclipse-pde/eclipse.pde/pull/1285 -->
+    <td class="title">New Modify button in Execution Environment section</td>
+    <td class="content">
+      <p>
+      	New 'Modify..' button is added in the Execution Environment section of the Overview tab of the Manifest Editor.
+      </p>
+    <p>
+      <img src="images/EESection_Manifest.png" />
+    </p>
```

Comment:
The HTML validation currently fails because the `alt` attributes is missing:
https://validator.w3.org/#validate_by_input
```suggestion
      <p>
        <img src="images/EESection_Manifest.png" alt="Modify button in Manifest Editor"/>
      </p>
```